uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/test/java/org/uberfire/ext/wires/core/grids/client/widget/grid/selections/impl/BaseCellSelectionManagerTest.java

```diff
@@ -544,6 +546,118 @@ public void adjustSelectionRightDownWithShiftKey() {
                      gridWidgetData.getSelectedCellsOrigin());
     }
 
+    @Test
+    public void testAdjustSelectionUpFromDataToHeader() {
+        cellSelectionManager.selectCell(0,
+                                        1,
+                                        false,
+                                        false);
+
+        cellSelectionManager.adjustSelection(SelectionExtension.UP, false);
+
+        final List<GridData.SelectedCell> selectedCells = gridWidgetData.getSelectedCells();
+        Assertions.assertThat(selectedCells).isEmpty();
```

Comment:
We'd normally have a static import in this scenario; however it's fine as it is really.